proof/refine/ARM/IpcCancel_R.thy

```diff
@@ -191,13 +191,45 @@ defs replyUnlink_assertion_def:
     \<equiv> \<lambda>replyPtr state s. state = BlockedOnReply (Some replyPtr)
                           \<or> (\<exists>ep d. state = BlockedOnReceive ep d (Some replyPtr))"
 
+lemma set_reply_obj_ref_valid_tcb[wp]:
```

Comment:
FIXME RT move, please. But it might be too much to label all the ainvs lemmas separately. Maybe you can put them together in one or a few places and label them.

Reply:
I don’t think we need to move them to AInvs at this point, actually. Just putting them together with the definition maybe would be good for now, I think.